notebooks/07_emg_hand_gesture_classification.ipynb

```diff
@@ -117,7 +117,8 @@
    "metadata": {},
```

Comment:
<p>NIT: Would it be possible to name the pipelines a bit more explicitly? "lr" can be confused with learning rate, ai with artificial intelligence, etc, for users who would read fast through the notebook ... maybe names like:</p><p>logistic_regr__eucl?</p><p>tangent_logistic_regr_affine_invariant?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/912/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>